org.gnome.glabels-3.json

```diff
@@ -155,18 +155,9 @@
             ],
             "sources": [
                 {
-                    "type": "git",
-                    "url": "https://github.com/jimevins/glabels.git",
-                    "tag": "glabels-3_4_1",
-                    "commit": "a6ba9388d9d22ca7be68c03a199f89725228a72f"
-                },
-                {
-                    "type": "patch",
-                    "path": "appdata.patch"
-                },
-                {
-                    "type": "patch",
-                    "path": "glabels-fixes.patch"
+                    "type": "archive",
+                    "url": "https://github.com/jimevins/glabels-qt/archive/refs/tags/glabels-3.99-master564.tar.gz",
```

Comment:
this is not a stable release.

Flathub is NOT for nightly release. Even less for switching like that.

Reply:
this release is the v4 RC…
no changes since them, seems not maintained any more
previous version 3.4.1 is really quite old…

Reply:
it is a different repository. So this should be submitted as a NEW app. Even according the the name it uses a different toolkit.